docs/metrics/bleu.md

```diff
@@ -1,23 +1,27 @@
----
-search:
-  exclude: true
----
-
 # BLEU
 
 !!! info inline end "BLEU vs. Precision"
 
     BLEU can be thought of as an analog to [precision](precision.md) for text comparisons.
 
-The BLEU (**B**i**L**ingual **E**valuation **U**nderstudy) score is a metric commonly used in a variety of NLP tasks ranging from Machine Translation to Text Summarization, to evaluate the quality of candidate texts. It quantifies the similarity between the candidate text and reference text as a score between 0 and 1 — 0 meaning there is no overlap with the ground truth text, and 1 meaning there is a perfect overlap. As the name suggests, it was originally developed for evaluating machine-translation models, but has since been adapted to many different tasks within NLP due to its dynamic nature for measuring textual similarity.
+The BLEU (**B**i**L**ingual **E**valuation **U**nderstudy) score is a metric commonly used in a variety of NLP tasks
+ranging from Machine Translation to Text Summarization, to evaluate the quality of candidate texts. It quantifies the
+similarity between the candidate text and reference text as a score between 0 and 1 — 0 meaning there is no overlap with
+the ground truth text, and 1 meaning there is a perfect overlap. As the name suggests, it was originally developed for
+evaluating machine-translation models, but has since been adapted to many different tasks within NLP due to its dynamic
+nature for measuring textual similarity.
 
 ## Implementation Details
 
 ### Background
 The BLEU score consists of two components — the Brevity Penalty and the n-gram Overlap.
 
-1. The **n-gram Overlap** counts the number of 1-grams, 2-grams, 3-grams, and 4-grams of the output text that match the 1-, ..., 4-grams in the reference text — which is analogous to a precision score for the text. The 1-gram precision ensures that the correct vocabulary is used, whereas the 4-gram precision ensures that the candidate text is coherent.
-2. The **Brevity Penalty** is applied to penalize the score for generating sentences that are less in length than the reference text. This is due to the fact that the n-gram Overlap precision tends to give disproportionately high values to candidate texts that are very short in length, but mostly contain n-grams in the reference text.
+1. The **n-gram Overlap** counts the number of 1-grams, 2-grams, 3-grams, and 4-grams of the output text that match the
+1-, ..., 4-grams in the reference text — which is analogous to a precision score for the text. The 1-gram precision
+ensures that the correct vocabulary is used, whereas the 4-gram precision ensures that the candidate text is coherent.
+2. The **Brevity Penalty** is applied to penalize the score for generating sentences that are less in length than the
+reference text. This is due to the fact that the n-gram Overlap precision tends to give disproportionately high values
+to candidate texts that are very short in length, but mostly contain n-grams in the reference text.
```

Comment:
nit for line 31: Is it cleaner to take the quarter root of the `i-gram Precision` rather than to the `1/4` power? Also if we do use `1/4`, would it make sense to use `\frac{1}{4}` instead of `1/4`? Currently looks like it could be related to the footnote.

Reply:
I would lean towards using `1/4` power rather than the quarter root because that exponent is equivalent to weight. 

> would it make sense to use \frac{1}{4} instead of 1/4?  
Yes, I think so. 

Updated:
<img width="678" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/7a80637b-58ba-411c-b4d7-073bfb9420f3">